dds/DCPS/security/AccessControl/LocalAccessCredentialData.cpp

```diff
@@ -6,125 +6,57 @@
 #include "LocalAccessCredentialData.h"
 #include "dds/DCPS/security/CommonUtilities.h"
 
-#include <fstream>
-
 OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL
 
 namespace OpenDDS {
 namespace Security {
 
 LocalAccessCredentialData::LocalAccessCredentialData()
 {
+
 }
 
 LocalAccessCredentialData::~LocalAccessCredentialData()
 {
+
 }
 
 bool LocalAccessCredentialData::load(const DDS::PropertySeq& props,
                                      DDS::Security::SecurityException& ex)
 {
-  const std::string file("file:");
-  bool permission = false, governance = false, ca = false;
+  std::string name, value;
```

Comment:
The previous change reduced the scope of `name` and `value` which also enables const.  The change proposed here undoes this.